content/App.tsx

```diff
@@ -50,10 +50,12 @@ const InnerApp: FC<AppProps> = ({ Component, pageProps }) => {
                 provider: provider,
                 signer: signer || undefined,
                 networkId,
+                synthetixjs,
```

Comment:
Also, since synthetixjs now just in `{contracts: ContractsMap}` we need to make sure that every hook + function in `@synthetixio/queries` only access `ctx.snxjs.contracts`. Did a quick search and found `ctx.snxjs.toBytes32` which would fail.

I think if we wanna do this change we should type the `createQueryContext` to accept `{contracts:`SynthetixJS['contracts']` instead

Reply:
☑️